deploy/kubernetes/Jenkinsfile.yml

```diff
@@ -18,5 +18,5 @@ stages:
             make check-no-changes;
         stage: lint
     k8sTest:
-        k8sTest: "v1.18.2,v1.17.2,v1.16.4,v1.15.7,v1.14.10"
+        k8sTest: "v1.20.7,v1.19.11,v1.18.2,v1.17.2,v1.16.4,v1.15.7,v1.14.10"
```

Comment:
It'd be nice to support 1.21 too :slightly_smiling_face:

Reply:
kind version required an update to work with 1.21